packages/remark-stringify/package.json

```diff
@@ -40,14 +41,16 @@
     "xtend": "^4.0.1"
   },
   "devDependencies": {
+    "dtslint": "^0.4.2",
```

Comment:
Install `typescript`

Reply:
Just curious, why? dtslint downloads and installs an array of TS versions just before tests?

Reply:
I prefer to install it to control typescript version in our workspace(vs-code). Otherwise, we always have to use releases with `next` tag. And sometimes, it shows false alarm in my experience.(It might be fine for now because typescript become quite stable than two years ago. But I don't want to take the risk.)

If you know some reliable articles against my comment, it should be nice time to discard `typescript` from `devDependencies` of other repos too. 😉

Reply:
If you're looking for early warnings w.r.t. type breakage, `dtslint` is probably your best bet, since it tests the minimum supported version of the types and all newer versions (including @next).

For projects that use the TS compiler API, I'll typically set up a CI build matrix against multiple TS versions (i.e., https://github.com/code-to-json/code-to-json/blob/2b46e9fb685f8a71c2d906d81a7c389d9b108f8e/.travis/setup-ts-version.sh#L4-L6 ) since sometimes those [breaking changes](https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes) are far more difficult to detect through conventional ts -> js usage.

If you want a local version of TS to allow vscode to work consistently, I'd suggest a separate PR with [a workspace config](https://github.com/code-to-json/code-to-json/blob/2b46e9fb685f8a71c2d906d81a7c389d9b108f8e/code-to-json.code-workspace#L3) that ensures all contributors benefit from this setup.

Reply:
> If you're looking for early warnings w.r.t. type breakage, dtslint is probably your best bet, since it tests the minimum supported version of the types and all newer versions (including @next).

I knew. But @next tagged releases has problems with vscode sometimes.

> If you want a local version of TS to allow vscode to work consistently, I'd suggest a separate PR with a workspace config that ensures all contributors benefit from this setup.

It doesn't fix my problem that coercing contributors to use same typescript version. 🙄 
We just make other users to use same typescript in their development environment for remark but not in their other projects. And installing it into `devDependencies` is the simplest solution as long as I know.